vmware_exporter/vmware_exporter.py

```diff
@@ -2041,6 +2132,20 @@ def main(argv=None):
     logging.info("Starting web server on port {address}:{port}".format(address=args.address, port=args.port))
     endpoint = endpoints.TCP4ServerEndpoint(reactor, args.port, interface=args.address)
     endpoint.listen(factory)
+
+    # If the reactor stops or hangs
+    if 'WATCHDOG_USEC' in os.environ:
```

Comment:
I like this mentality, but how would this impact docker installs?

Reply:
I am honestly not sure how this would impact use in docker. In my enviroment we had some flaky links and it was causing the whole process to hang.

Reply:
I did a quick and dirty rebase to have master skip the commit with the systemd changes. The systemd changes are in https://github.com/Z903/vmware_exporter/tree/systemd.

Reply:
Thanks